src/System/OS.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Dock\System;
+
+class OS
+{
+    const MAC = 1;
+    const LINUX = 2;
+    const WINDOWS = 3;
+
+    /**
+     * @return int Current Operating System
+     */
+    public function get()
```

Comment:
This method is only used in the task provider right now. I'd move this method to private and create two `isLinux` and `isMac` public methods.